pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -14,6 +14,9 @@
 # Kernel dependencies
 , kernel ? null
 , enablePython ? true
+
+# for determining the latest compatible linuxPackages
+, pkgs
```

Comment:
Perhaps just declare the kernels explicitly here, shouldn't be much extra work (search & replace)

Reply:
I think that makes a lot of sense if it was expected to be overriden, but in this case it should not be touched by out-of-tree code

Reply:
then again, it's only one line of code difference

Reply:
applied